lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -782,6 +800,191 @@ setNonEnumerableReadOnly( REPL.prototype, 'clearUserDocs', function clearUserDoc
 	return this;
 });
 
+/**
+* Gets all available themes in the syntax-highlighter.
```

Comment:
We should update the examples and descriptions based on what gets changed in the README